backend/src/agents/__init__.py

```
@@ -150,6 +152,41 @@ async def init_subagents(
     return result
 
 
+async def init_skills_as_subagents(
+    skill_service: SkillService,
+    service_context: ServiceContext,
+    categories: list[str] = None,
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)